src/SourceNodes/sourcenode.js

```diff
@@ -371,8 +377,11 @@ class SourceNode extends GraphNode {
         this._triggerCallbacks("render", currentTime);
 
         if (currentTime < this._startTime) {
-            clearTexture(this._gl, this._texture);
-            this._state = STATE.sequenced;
+            // A quick check to ensure we don't call 'clearTexture' every RAF when only needed once
+            if (this._state != STATE.sequenced) {
+                clearTexture(this._gl, this._texture);
```

Comment:
Would the texture still be cleared be cleared if you played past the startTime and then seeked back before it?